### PR TITLE
Change docker auth properties 

### PR DESCRIPTION
**Pull request by mmorhun**
Signed-off-by: Mykola Morhun <mmorhun@codenvy.com>

### CODE REVIEW

**Comment by mmorhun** [comment]
@skabashnyuk ?

**Comment by mmorhun** [comment]
@garagatyi ?

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties [hunk 1]
```diff
@@ -70,11 +70,11 @@ schedule.core_pool_size=10
 ### that are constructed when the workspace is started. The images used to generate containers
 ### for the machines can come from DockerHub or a private Docker registry.
 machine.docker.registry=${CHE_REGISTRY_HOST}:5000
+machine.docker.snapshot.registry_namespace=NULL
```
Comment (garagatyi) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties]: Add comment

**Comment by garagatyi** [comment]
LGTM

**Comment by skabashnyuk** [comment]
ok

**Pull request merged**